docker-compose.test.yml

```diff
@@ -12,8 +12,12 @@ services:
       - ATO_PRIVATE_KEY_FILE=../../shared_files/ATO_private.pem
       - OKTA_CLIENT_ORGURL=https://cms-sandbox.oktapreview.com
       - OKTA_EMAIL=shawn@bcda.aco-group.us
-# Add the Okta token below and uncomment
+# Add the Okta values below and uncomment
+#     - BCDA_AUTH_PROVIDER=okta
 #     - OKTA_CLIENT_TOKEN=<token>
+#     - OKTA_OAUTH_SERVER_METADATA=
```

Comment:
It looks like we can remove `OKTA_OAUTH_SERVER_METADATA` here and in docker-compose.yml, right? Since the URL is being created using the base URL + server ID.